packages/lodestar/src/network/encoder.ts

```diff
@@ -55,12 +55,12 @@ export class ReqRespEncoder {
               return encoder.encode(type, result);
             }, chunk.output) as Buffer;
           }
-          let status = 0;
-          if(chunk.err) {
-            status = chunk.err.status === ERR_INVALID_REQ ? 1 : 2;
-            //TODO: stop yielding
-          }
+          const status = chunk.err && chunk.err.status;
```

Comment:
ain't this gonna be true/false now?

Reply:
no, it's gonna be `undefined` or  `typeof chunk.err.status` which is number. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators